src/users/users-module.ts

```diff
@@ -0,0 +1,39 @@
+import { randomUUID } from "crypto";
+import PostgresDB from "../modules/psql";
+
+class Users extends PostgresDB {
+    constructor(connectionString: string) {
+        super(connectionString);
+    }
+
+    
+    async addUser(username: string, email: string): Promise<void> {
+      // USER_TABLE environment variable contains the name of the table meant for storing user information
+      const id = randomUUID();
+      const userTable = process.env.USER_TABLE;
```

Comment:
Would it be better to have userTable initialized when constructing the user class above?

Reply:
done